### DISCUSSION TITLE
Suggestion: Acomplishments/Achievements

### DISCUSSION THREAD

**Discussion by FormBurden** [voxeltycoon/issues#804] [General]
Hello, got to say, this is a fun and addictive game. Love it!

Anyway, as I've noticed there is a lot of stat/money tracking and such, and I was wondering about possibly in a future patch add accomplishments/achievements to the game. 

Now, I know for a lot of developers Steam achievements usually come at some later time, and/or end of development, which sure, do that. While Steam achievements are great, they're a one time thing (usually). And I usually love it when games have an in-game/character specific accomplishments/achievements. Like Factorio (which mods can add more and such). 

i.e Carry a total of 1m tons of cargo. Earn 10m money (30, 100, etc.) Extract X amount of items from the resources, etc. 

Things you can look forward to on your playthrough, not exactly just a one-time achievement/accomplishment.

And like Factorio possibly add it so mods can take advantage of creating their own achievements/accomplishments.

What are your thoughts?

**Comment by DanielBarreleiro** [comment]
@FormBurden 
Achievements are in the roadmap:
https://trello.com/c/LlIv3uIb

**Reply by FormBurden** [reply]
Got it. Nice. I mean like I said, I hope it's more than just Steam achievements and something in-game as well. Makes progression more fun.

**Reply by DanielBarreleiro** [reply]
There's quests and more too, check the roadmap, its public.

**Reply by FormBurden** [reply]
Yeah I did see. Can't wait.
